src/ui/Button/styles.ts

```diff
@@ -11,10 +11,12 @@ export const styles = StyleSheet.create({
   },
   primary: {
     backgroundColor: palette.Common.white,
-    borderColor: 'rgba(29, 33, 42, 0.16)',
+    borderColor: 'rgba(255, 255, 255, 1)',
```

Comment:
Someday we will have to transfert this color into the palette or into a theme dedicated file

Reply:
@Ldoppea Absolutely, and this applies to many other values. It was intended that the current feature with "dynamic theming" would rework these values and erase any magic number used in the app.